internal/benchmarks/src/config.ts

```diff
@@ -0,0 +1,11 @@
+import { DEVNET_NETWORK_URL } from '@internal/utils';
+import { TransactionType } from 'fuels';
+
+export const DEVNET_CONFIG = {
```

Comment:
Could we merge the `e2e` configs here and import to `fuel-gauge`?

Reply:
I've removed this all together aea62fb1f